apis/api.ts

```diff
@@ -10,11 +11,36 @@ const API = axios.create({
   },
   withCredentials: true,
 })
+API.interceptors.request.use(async (config) => {
+  const tokenManager = new TokenManager()
+
+  if (
+    !tokenManager.validateToken(
+      tokenManager.accessExpiredAt,
+      tokenManager.accessToken
+    ) &&
+    tokenManager.validateToken(
+      tokenManager.refreshExpiredAt,
+      tokenManager.refreshToken
+    )
+  ) {
+    tokenManager.reissueToken()
+  } else if (
+    !tokenManager.validateToken(
+      tokenManager.accessExpiredAt,
+      tokenManager.accessToken
+    ) &&
+    !tokenManager.validateToken(
+      tokenManager.refreshExpiredAt,
+      tokenManager.refreshToken
+    )
+  )
```

Comment:
```suggestion

  let accessTokenIsValid = tokenManager.validateToken(
    tokenManager.accessExpiredAt,
    tokenManager.accessToken
  )
  let refreshTokenIsValid = tokenManager.validateToken(
    tokenManager.refreshExpiredAt,
    tokenManager.refreshToken
  )
  if (!accessTokenIsValid && refreshTokenIsValid) {
    tokenManager.reissueToken()
  } else if (accessTokenIsValid! && !refreshTokenIsValid)
```
사소하긴한데 이런식으로 변수로 빼는거 어떤가요?

Reply:
좋은 방식 같아요! 고려해보겠습니다.

Reply:
[065a539](https://github.com/GSM-MSG/Hi-v2-FrontEnd/pull/108/commits/065a53957456b8d0fcf7ae48b7c8db2936131f9b)
반영했습니다